Sources/Core/Color/ColorTheme.swift

```diff
@@ -14,33 +14,38 @@ public struct ColorTheme {
         nilLiteral: { $0 },
         boolLiteral: { $0 },
         stringLiteral: { $0 },
-        numberLiteral: { $0 }
+        numberLiteral: { $0 },
+        urlLiteral: { $0 }
     )
     public static let `default` = ColorTheme(
         typeName: { $0.yellow() },
         nilLiteral: { $0.red() },
         boolLiteral: { $0.blue() },
         stringLiteral: { $0.cyan() },
-        numberLiteral: { $0.green() }
+        numberLiteral: { $0.green() },
+        urlLiteral: { $0.blue().underline() }
```

Comment:
URL is not **Literal** in Swift.
Therefore I suggest to change name to simply `url`.

Reply:
Indeed!

I fixed in 67891f2.